app/gateway-manager/data-plane-reference.md

```diff
@@ -13,24 +12,213 @@ works_on:
 api_specs:
     - konnect/control-planes-config
 faqs:
-  - q: Will a new serverless gateway be provisioned in the same region as {{site.konnect_short_name}}?
-    a: Deployment on the same region is not guaranteed. 
+  - q: How can I verify that a Data Plane node is running?
+    a: |
+      You can verify a Data Plane node by accessing a configured route through its proxy URL. 
+      By default, {{site.base_gateway}} listens on port `8000`, so a request to `http://localhost:8000/{your-route}` (or your custom hostname) 
+      should return the expected response from your upstream service.
+
+  - q: How do I access services through a Data Plane node running on Kubernetes?
+    a: |
+      1. Run the following command to get the external IP and port:
+         ```bash
+         kubectl get service my-kong-kong-proxy -n kong
+         ```
+      2. Find the IP in the `EXTERNAL-IP` column and use it with port `80` or `443` along with your route.
+
+         For example, if the external IP is `35.233.198.16` and your route is `/mock`, access your service at:
+         ```
+         http://35.233.198.16:80/mock
+         ```
+  - q: Can I choose a specific {{site.base_gateway}} version when using Quickstart scripts in Gateway Manager?
+    a: |
+      Yes. Gateway Manager allows you to select the {{site.base_gateway}} version for your Quickstart scripts—except when using cloud provider scripts for AWS, Azure, and GCP.
+
+  - q: Can I SSH directly into Konnect Data Plane nodes?
+    a: |
+      No. Direct SSH access is not possible because the SSH keys are randomly generated and not exposed. 
+      To access nodes, use the cloud provider’s tools:
+      * [AWS EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-methods.html)
+      * [Azure Cloud Shell](https://learn.microsoft.com/azure/cloud-shell/overview)
+      * [Google Cloud SSH](https://cloud.google.com/compute/docs/instances/ssh)
```

Comment:
```suggestion
  - q: Can I SSH directly into Konnect Data Plane nodes?
    a: |
      No. Direct SSH access is not possible because the SSH keys are randomly generated and not exposed. 
      If you're running Data Plane nodes in a cloud provider, use the cloud provider’s tools to access the nodes:
      * [AWS EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-methods.html)
      * [Azure Cloud Shell](https://learn.microsoft.com/azure/cloud-shell/overview)
      * [Google Cloud SSH](https://cloud.google.com/compute/docs/instances/ssh)
```